README.md

```diff
@@ -0,0 +1,16 @@
+# Portfolio (STEP Project)
+
+## Introduction
+This is the repository hosting the files for the repository I built as my first project for the STEP program!
+Once deployed, I will attach a URL at which it can viewed.
+
+## Screenshots
```

Comment:
nit: don't leave empty sections. you can always add them up with contexts later, but it's very easy to forget about them and never ever touch them in the future.

Reply:
Thanks for the feedback. I'll go ahead and remove the section until I have the content to add.